packages/hooks/src/use-toggle-state/use-toggle-state.ts

```diff
@@ -1,10 +1,12 @@
 import { useCallback, useState } from 'react';
 
-const useToggleState = (defaultValue?: boolean) => {
+const useToggleState = (
+  defaultValue?: boolean
+): [boolean, (forceIsToggled?: boolean) => void] => {
```

Comment:
it's better to be explicit here rather than moving the `ReturnType` to a declaration.
We need to do this, because TS misunderstood the return type 

output of intellisense, note the `[]`
```
const useToggleState: (defaultValue?: boolean | undefined) => (boolean | ((forceIsToggled?: boolean | undefined) => void))[] // <-- note the []
```